requirements.txt

```diff
@@ -0,0 +1,40 @@
+astroid==3.2.4
+backports.tarfile==1.2.0
+build==1.2.1
+certifi==2024.7.4
+charset-normalizer==3.3.2
+dill==0.3.8
+docutils==0.21.2
+flake8==7.1.0
+idna==3.7
+importlib_metadata==8.0.0
+isort==5.13.2
+jaraco.classes==3.4.0
+jaraco.context==5.3.0
+jaraco.functools==4.0.1
+keyring==25.2.1
+markdown-it-py==3.0.0
+mccabe==0.7.0
+mdurl==0.1.2
+more-itertools==10.3.0
+nh3==0.2.18
+packaging==24.1
+pkginfo==1.10.0
+platformdirs==4.2.2
+prsdk==0.0.1
```

Comment:
We don't need all these requirements. Surely not `prsdk==0.0.1`
I think we only need:
```
flake8==7.1.0
pylint==3.2.6
```

Reply:
Trimmed requirements down to these 2